tests/library/dbg/tests/test_context_commands.py

```diff
@@ -43,7 +43,7 @@ async def test_context_disasm_show_fd_filepath(ctrl: Controller) -> None:
 
     line_call_read, line_fd, line_buf, line_nbytes, *_rest = lines_after_call_read
 
-    assert "call   read@plt" in line_call_read or "call   read" in line_call_read
+    assert "read@plt" in line_call_read or "read" in line_call_read
```

Comment:
```suggestion
    assert "read" in line_call_read
```
The other part doesn't make sense since its a subset